config/settings.yml

```diff
@@ -1 +1,3 @@
-basic_auth_required: true
\ No newline at end of file
+basic_auth_required: true
+features:
+  home_text: false
```

Comment:
could we add an inline comment to say what the purpose is of this config setting

Reply:
done https://github.com/DFE-Digital/register-trainee-teacher-data/pull/44/commits/88429b4afbb13f436fdf818a6b1a2029536d9957